src/ex3/mondayu-logger-todoapp.js

```diff
@@ -0,0 +1,82 @@
+import fs from "fs";
+
+const args = process.argv;
+const currentDirectory = process.cwd() + "\\";
```

Comment:
in linux it should be 
```suggestion
const currentDirectory = process.cwd() + "/";
```

(I needed to change it in order to make it work in my mac)
It's better to use nodejs built in methods in order to not to deal with different environments edge cases
https://nodejs.org/api/path.html#pathresolvepaths